operator/src/test/java/org/keycloak/operator/testsuite/unit/CRSerializationTest.java

```diff
@@ -28,8 +28,7 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.notNullValue;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please, don't use the star imports.

Reply:
@mabartos I got rid of the start import.